imcsdk/imcsession.py

```diff
@@ -391,6 +391,20 @@ def _refresh(self, auto_relogin=False):
         self.__start_refresh_timer()
         return True
 
+    def __is_fabric_interconnect(self):
+        from .imcmethodfactory import config_resolve_class
+
+        nw_elem = config_resolve_class(cookie=self.__cookie,
+                                       class_id="networkElement")
+        try:
+            nw_elem_response = self.post_elem(nw_elem)
+            if nw_elem_response.error_code == 0:
+                return False
```

Comment:
@prvashis so `__is_fabric_interconnect` should return `True` when it is a FI otherwise `False`. Am I reading the code wrong? It seems to be opposite